apps/stable_diffusion/shark_sd.spec

```diff
@@ -19,6 +19,7 @@ datas += copy_metadata('importlib_metadata')
 datas += copy_metadata('torch-mlir')
 datas += copy_metadata('omegaconf')
 datas += copy_metadata('safetensors')
+datas += copy_metadata('Pillow')
```

Comment:
this is already in tree ?

Reply:
I did a git pull before pushing, realized later I should not have done that. So I got 2 commits from Ean and Abhishek